### PR TITLE
Made input boxes for the pages 

### PR DESCRIPTION
**Pull request by ellietms**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-grants-productivity-web",
-  "version": "1.13.2",
+  "version": "1.14.2",
```
Comment (diya2214) [inline package.json:3]: please make this "version": "1.14.0"

File: app/templates/page.njk [hunk 2]
```diff
@@ -37,11 +37,17 @@
...
-        
+
         <div class="govuk-!-padding-top-5">
+        {%if backLink !== 'slurry-currently-treated' %}
         {{ govukButton({
           text: "Continue"
         }) }}
+        {% else %}
+        {{ govukButton({
+          text: "Getyourscore"
```
Comment (diya2214) [inline app/templates/page.njk:48]: spaces **Get your score**

File: app/routes/index.js [hunk 3]
```diff
@@ -65,12 +65,26 @@ const checkBoxes = (data, question) => {
...
+}
+
 const getOptions = (data, question) => {
   switch (question.type) {
     case 'single-answer':
       return radioButtons(data, question)
     case 'multi-answer':
       return checkBoxes(data, question)
+    case 'input':
+      return inputBoxes(question)
```
Comment (diya2214) [inline app/routes/index.js:87]: i think we have to pass data as well so we display the user data if the user is revisiting

**Comment by ellietms** [comment]
Not needed

**Pull request closed without merging**